recipes/lidar/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lidar" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0f6059a915c2f20668e74e347592171c3f897be90e69aea5ccf2c8e7707e1ec3
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
`noarch` recipes are built on Linux only, so you can ignore the Windows/OSX failures. Thus, this recipe can be `noarch: python` and you can remove the `skip` section. If some `run` dependencies are missing for Windows, the user can additionally install them, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

Reply:
So I just revert to the previous commit?

Reply:
Yes please.